.idea/libraries/jetty.xml

```diff
@@ -1,7 +1,7 @@
 <component name="libraryTable">
   <library name="jetty">
     <CLASSES>
-      <root url="jar://$PROJECT_DIR$/build/third_party/java/jetty/jetty-repacked.jar!/" />
+      <root url="jar://$PROJECT_DIR$/third_party/java/jetty/jetty-repacked.jar!/" />
```

Comment:
this file also doesn't exist until you build it... the build folder one should exist for you, not sure why it doesn't